### ISSUE TITLE
[BUG][WEBJS] - GitHub Connection Error in Webhook Configuration: ECONNREFUSED When Fetching Version Information

### ISSUE THREAD

**Issue by UnitedWoods** [allburov/whatsapp-http-api#316]
**Describe the bug**

I can't authenticated the session, when I scan the QR code it will failed:

An error occurs when configuring and sending Webhook messages, specifically when attempting to fetch version information from the wppconnect-team GitHub repository. The error message indicates a failure to connect to GitHub, with the ECONNREFUSED error being returned.

**Version**

```json
{
  "version": "2024.5.2",
  "engine": "WEBJS",
  "tier": "CORE",
  "browser": "/usr/bin/chromium"
}
```

**To Reproduce** Steps to reproduce the behavior:

1. Go to '/api/sessions/start'
2. See error

**Expected behavior**

Want to authenticated the session:
[WhatsappSession - default] Session 'default' has been authenticated!

**Engine** I'm using **WEBJS** engine.

> If you set `WHATSAPP_DEFAULT_ENGINE` explicitly to
> [one of the engine](https://waha.devlike.pro/docs/how-to/engines/) - please
> provide that information here and in the issue title.

**Requests - Responses**

> Please attach all sent requests, responses that you sent.

**Docker Logs**

2024-05-04 13:36:45 [Nest] 30  - 05/04/2024, 5:36:45 AM     LOG [SessionManager] 'default' - starting session...
2024-05-04 13:36:45 [Nest] 30  - 05/04/2024, 5:36:45 AM     LOG [Webhook - default] Configuring webhooks for https://httpbin.org/post...
2024-05-04 13:36:45 [Nest] 30  - 05/04/2024, 5:36:45 AM     LOG [Webhook - default] Event 'session.status' is enabled for url: https://httpbin.org/post
2024-05-04 13:36:45 [Nest] 30  - 05/04/2024, 5:36:45 AM     LOG [Webhook - default] Webhooks were configured for https://httpbin.org/post.
2024-05-04 13:36:45 [Nest] 30  - 05/04/2024, 5:36:45 AM     LOG [Webhook - default] Event 'message' is enabled for url: https://httpbin.org/post
2024-05-04 13:36:45 [Nest] 30  - 05/04/2024, 5:36:45 AM     LOG [Webhook - default] Sending POST to https://httpbin.org/post...
2024-05-04 13:36:46 Error fetching version 2.2412.54 from remote FetchError: request to https://raw.githubusercontent.com/wppconnect-team/wa-version/main/html/2.2412.54.html failed, reason: 
2024-05-04 13:36:46     at ClientRequest.<anonymous> (/app/node_modules/node-fetch/lib/index.js:1501:11)
2024-05-04 13:36:46     at ClientRequest.emit (node:events:518:28)
2024-05-04 13:36:46     at ClientRequest.emit (node:domain:488:12)
2024-05-04 13:36:46     at TLSSocket.socketErrorListener (node:_http_client:500:9)
2024-05-04 13:36:46     at TLSSocket.emit (node:events:518:28)
2024-05-04 13:36:46     at TLSSocket.emit (node:domain:488:12)
2024-05-04 13:36:46     at emitErrorNT (node:internal/streams/destroy:169:8)
2024-05-04 13:36:46     at emitErrorCloseNT (node:internal/streams/destroy:128:3)
2024-05-04 13:36:46     at process.processTicksAndRejections (node:internal/process/task_queues:82:21) {
2024-05-04 13:36:46   type: 'system',
2024-05-04 13:36:46   errno: 'ECONNREFUSED',
2024-05-04 13:36:46   code: 'ECONNREFUSED'
2024-05-04 13:36:46 }

**Additional context**

If I run it on a Linux system on a remote server, it works fine, but if I run it locally (Mac OS arm), I will report an error.

**Comment by allburov** [comment]
That's weird, why we didn't inline it in the project itself :D 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Fixed in `2024.5.4`

If it doesn't help - try to use a different image, chrome based one
https://waha.devlike.pro/docs/how-to/engines/#docker-images
```
devlikeapro/whatsapp-http-api-plus:chrome
```


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
